ads/caprofitx.md

```diff
@@ -28,13 +32,40 @@ limitations under the License.
 </amp-ad>
 ```
 
-## Configuration
-For configuration details, please email ca_profitx_support@cyberagent.co.jp.
+### Ad placement with `mediaid`
+
+```html
+<amp-ad
+  width="320"
+  heigth="100"
+  type="caprofitx"
+  data-mediaid="3752"
+  data-tag-places="10007">
+</amp-ad>
+```
+
+## Configurations
+
+For details on the configuration semantics and to generate your tags,
+please email ca_profitx_support@cyberagent.co.jp.
+
+### (Deprecated) Ad placement with `tagid`
+
+Supported parameters:
 
-### Required parameters
+| Parameter    | Required |
+|--------------|--------- |
+| data-tagid   | Yes      |
```

Comment:
Can I ask what's your deprecation plan for `data-tagid`. Are you going to make ads with `data-tagid` invalid in the future? Thank you

Reply:
Thank you for your confirmation.

Currently, a part of our customers are still using `data-tagid` field. 
We are trying to contact them about the replacement with `data-mediaid` but it may take time.
After all inventories are replaced I will make `data-tagid` invalid.

Btw, I want to mention that new customer should not use `data-tagid`.
Should I delete `(Deprecated) Ad placement with tagid` section?

Reply:
@zhouyx 
Thank you for your confirmation.

Currently, a part of our customers are still using `data-tagid` field. 
We are trying to contact them about the replacement with `data-mediaid` but it may take time.
After all inventories are replaced I will make `data-tagid` invalid.

Btw, I want to mention that new customer should not use `data-tagid`.
Should I delete `(Deprecated) Ad placement with tagid` section?

Reply:
Thank you for the detailed explanation. Yes I agree in that case maybe it's best to remove the example section.